homeassistant/components/shelly/const.py

```diff
@@ -1,5 +1,7 @@
 """Constants for the Shelly integration."""
 
 COAP_CONTEXT = "coap_context"
+CONFIG_ENTRY = "config_entry"
 DOMAIN = "shelly"
+GENERAL = "general"
```

Comment:
Should we prefix the constant names to indicate _what_ is "config entry" or "general" ?

Reply:
Great idea, maybe we can use `DATA_GENERAL` and `DATA_CONFIG_ENTRY`. What do you think?

Reply:
Sounds good!